src/test/java/LoginTest.java

```diff
@@ -0,0 +1,114 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page.HomePage;
+//import page.LoginPage;
+import pageFactory.LoginPage;
+
+public class LoginTest extends BaseTest {
+
+
+@Test
+    public void loginEmptyEmailPassword()  {
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.provideEmail(" ").provideEmail("jKV0uSX6z1dv").clickSubmit();
+
+        String expectedURL= "https://qa.koel.app/";
+        Assert.assertEquals(driver.getCurrentUrl(), expectedURL);
+
+        driver.quit();
+        /*
+        String expectedURL= "https://qa.koel.app/";
+        //navigateToPage();
+        enterEmail("");
+        enterPassword("jKV0uSX6z1dv");
+
+ */
+    }
+@Test
+
+    public void loginValidEmailPassword()  {
+
+   LoginPage loginPage = new LoginPage(driver);
+   HomePage homePage = new HomePage(driver);
+
+   // login has step in it with a method
+
+loginPage.provideEmail("christina.taylor@testpro.io").providePassword("jKV0uSX6z1dv").clickSubmit();
+
+ Assert.assertTrue(homePage.getUserAvatarIcon().isDisplayed());
+
+
+
+
+
+
+
+
+
+
+    }
+
+@Test
+
+    public void loginInvalidEmailValidPassword()  {
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+      loginPage.provideEmail("invalidemail@testpro.io").providePassword("jKV0uSX6z1dv").clickSubmit();
+
+    String expectedURL= "https://qa.koel.app/";
+
+
+        Assert.assertEquals(driver.getCurrentUrl(), expectedURL);
+
+    }
+
+@Test
+
+    public void loginEmailEmptyPassword()  {
+
+
+
+     LoginPage loginPage = new LoginPage(driver);
+     HomePage homePage =new HomePage(driver);
+
+     loginPage.provideEmail("christina.taylor@testpro.io").providePassword(" ").clickSubmit();
+
+
+
+        String expectedURL= "https://qa.koel.app/";
+        Assert.assertEquals(driver.getCurrentUrl(), expectedURL);
+
+
+    //  enterEmail("christina.taylor@testpro.io");
+
+
+    }
+    @Test( dataProvider ="NegativeLoginTestData", dataProviderClass = TestDataProvider.class)
+    public void loginNegativeTest(String email, String password) throws InterruptedException {
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+
+        loginPage.login();
+
+        String expectedURL= "https://qa.koel.app/";
+        Assert.assertEquals(driver.getCurrentUrl(),expectedURL);
+
+         /*  enterEmail(email);
+        enterPassword(password);
+        submit();
+
+       */
+
+
+    }
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)